src/Websites/Websites/Cmdlets/AppServicePlans/NewAzureAppServicePlan.cs

```diff
@@ -63,6 +64,9 @@ public class NewAzureAppServicePlanCmdlet : AppServicePlanBaseCmdlet
         [Parameter(Mandatory = false, HelpMessage = "Run cmdlet in the background")]
         public SwitchParameter AsJob { get; set; }
 
+        [Parameter(ParameterSetName = ParameterSet1Name, Position = 7, Mandatory = false, HelpMessage = "Tags are name/value pairs that enable you to categorize resources")]
+        public IDictionary<string, string> Tags { get; set; }
```

Comment:
Parameter type shpould be Hashtable

Reply:
As the Tags parameter type is Dictionary in Site Config,we declared it as the same.

Reply:
The standard for this is to use Hashtable and convert to the wire type (usually ```Dictionary<string, string>```).  Please adhere to the standard